tensorrt_llm/_torch/attention_backend/sparse/minimax_m3/triton_metadata.py

```
@@ -674,242 +651,222 @@ def build_runtime_metadata_from_kv_manager(
+            return
+        num_contexts = int(getattr(self, "num_contexts", 0) or 0)
+        batch_size = int(seq_lens.shape[0])
+        if batch_size == 0:
+            return
+
+        # The cache device hosts every paged buffer; this is the
+        # device the forward path consumes.
+        try:
+            layer_buf = kv_cache_manager.get_buffers(0)
+            cache_device = layer_buf.device
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)